### PR TITLE
Refactor awssecurityhub and add endpoint

### PR DESCRIPTION
**Pull request by manuel-sommer**
-  refactor awssecurityhub
-  add endpoint to Inspector
-  add endpoint to GuardDuty

### CODE REVIEW

**Comment by manuel-sommer** [comment]
@testaccount90009, fyi.

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Comment by manuel-sommer** [comment]
close and reopen to retrigger pipeline

File: dojo/tools/awssecurityhub/guardduty.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                mitigated = datetime.utcnow()
+        description = f"This is a GuardDuty Finding\n{finding.get('Description', '')}"
+        description += f"SourceURL: {finding.get('SourceUrl', '')}\n"
+        description += f"AwsAccountId: {finding.get('AwsAccountId', '')}\n"
+        description += f"Region: {finding.get('Region', '')}\n"
+        title_suffix = ""
+        hosts = list()
+        for resource in finding.get("Resources", []):
+            component_name = resource.get("Type")
+            print("/777777777777777777777")
```
Comment (blakeaowens) [inline dojo/tools/awssecurityhub/guardduty.py:40]:
```suggestion
```

File: dojo/tools/awssecurityhub/guardduty.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            if component_name in ("AwsEcrContainerImage", "AwsEc2Instance"):
+                print("AFJIEOFJIOEFJOFJIO")
```
Comment (blakeaowens) [inline dojo/tools/awssecurityhub/guardduty.py:42]:
```suggestion
```

**Review by blakeaowens — CHANGES_REQUESTED** [review]
Please remove print statements.

**Comment by manuel-sommer** [comment]
Done @blakeaowens

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**